.github/workflows/e2e-metaprotocol-gateway.yaml

```diff
@@ -18,7 +18,7 @@ on:
     - "**/*.md"
 
 env:
-  ISTIO_VERSION: 1.18.2
+  ISTIO_VERSION: 1.18.1
```

Comment:
I'm curious about the reason for downgrading the version.

Reply:
btw, There are so many branches here

<img width="328" alt="image" src="https://github.com/aeraki-mesh/aeraki/assets/19339970/9058b197-633a-4aa0-8228-145e9a20df25">

Reply:
> ut the reason for downgrading the version.

The envoy commit for Istio 1.8.2 doesn't exist for some reason.  So I decided to use 1.8.1, we can upgrade to a later release when they come out.

https://github.com/istio/proxy/blob/3c27a1b0cf381ca854ccc3a2034e88c206928da2/WORKSPACE#L38

```bash
(base) ➜  envoy git:(main) g checkout f826f1dbddd3326abee19009ea9f019e8a9d5997    
fatal: reference is not a tree: f826f1dbddd3326abee19009ea9f019e8a9d5997
```

Reply:
> btw, There are so many branches here
> 
> <img alt="image" width="328" src="https://user-images.githubusercontent.com/19339970/261481612-9058b197-633a-4aa0-8228-145e9a20df25.png">

Those are development branches and can be deleted.